src/Invariants/GraphRules/domination_rule.jl

```diff
@@ -4,6 +4,6 @@ function apply!(
     g::AbstractGraph,
 )
     for v in copy(target_set)  # use copy to avoid modifying set during iteration
-        union!(target_set, neighbors(g, v))
+        union!(target_set, Graphs.neighbors(g, v))
```

Comment:
@joshyaffee Notice that in Julia you to not need to specify the namespace from which a function comes from. So, my old code using `graphs(g, v)` which comes from `using Graphs` was completely valid. However, since you are new to Julia, and Julia packages like `Graphs.jl`, _I decided to change the code to explicitly state which namespace a given function belongs to_, i.e., `Graphs.neighbors(g, v)` instead of `neighbors(g, v)`, or `Graphs.degree(g)` instead of `degree(g)`. I hope that this helps you learn where the functions come from